homeassistant/components/nut/__init__.py

```diff
@@ -26,22 +26,30 @@
 from homeassistant.helpers.update_coordinator import DataUpdateCoordinator, UpdateFailed
 
 from .const import (
-    COORDINATOR,
     DEFAULT_SCAN_INTERVAL,
     DOMAIN,
     INTEGRATION_SUPPORTED_COMMANDS,
     PLATFORMS,
-    PYNUT_DATA,
-    PYNUT_UNIQUE_ID,
-    USER_AVAILABLE_COMMANDS,
```

Comment:
Can we remove them from `.const` now?

Reply:
valid point ... just forgot them 🙈

would be great if we could have a check for unused (_nowhere imported_) constants 🤔